src/System.Runtime/tests/System/StringTests.cs

```diff
@@ -435,21 +429,12 @@ public static void Concat_Invalid()
         }
 
         [Theory]
-        [InlineData("Hello", 0, 0, 5, new char[] { 'H', 'e', 'l', 'l', 'o' })]
-        [InlineData("Hello", 1, 5, 3, new char[] { '\0', '\0', '\0', '\0', '\0', 'e', 'l', 'l', '\0', '\0' })]
-        [InlineData("Hello", 2, 0, 3, new char[] { 'l', 'l', 'o', '\0', '\0', '\0', '\0', '\0', '\0', '\0' })]
-        [InlineData("Hello", 0, 7, 3, new char[] { '\0', '\0', '\0', '\0', '\0', '\0', '\0', 'H', 'e', 'l' })]
-        [InlineData("Hello", 5, 10, 0, new char[] { '\0', '\0', '\0', '\0', '\0', '\0', '\0', '\0', '\0', '\0' })]
-        [InlineData("H" + SoftHyphen + "ello", 0, 0, 3, new char[] { 'H', '\u00AD', 'e' })]
+        [MemberData("CopyTo_TestData", MemberType = typeof(StringTestsSpan))]
```

Comment:
@ahsonkhan is this correct?

Reply:
Yes. Looks correct to me.

However, instead of having separate classes, what if we keep one class - maybe StringAndSpanTests (partial and in two files). This way, we can just do `[MemberData("CopyTo_TestData")]` here.

Reply:
@tarekgh, would you be fine with the change to rename this string test class?

Reply:
>However, instead of having separate classes, what if we keep one class - maybe StringAndSpanTests (partial and in two files). This way, we can just do [MemberData("CopyTo_TestData")] here.

I agree...wait.

Reply:
@ahsonkhan @tarekgh i added base class with tests data...you like.